pallets/token-manager/src/lib.rs

```diff
@@ -596,17 +597,19 @@ impl<T: Config> Pallet<T> {
         token_id: &T::TokenId,
         amount: &T::TokenBalance,
         sender_nonce: u64,
-    ) -> Vec<u8> {
-        return (
-            SIGNED_TRANSFER_CONTEXT,
-            proof.relayer.clone(),
-            from,
-            to,
-            token_id,
-            amount,
-            sender_nonce,
+    ) -> BoundedVec<u8, MaximumSignedParamsSize> {
+        return BoundedVec::truncate_from(
```

Comment:
Is this safe? Does the proof have a fixed size?

Reply:
the tests are passing with the `pub type MaximumSignedParamsSize = ConstU32<256>;` constant

Reply:
what other testing should be done here?